src/Utils/dialogUtils.ts

```diff
@@ -210,3 +211,49 @@ export function findMatchingTrainDialog(trainDialog: CLM.TrainDialog, trainDialo
     }
     return null
 }
+
+// Returns true is trainDialog1 is longer than trainDialog2
+export function isLonger(trainDialog1: CLM.TrainDialog, trainDialog2: CLM.TrainDialog): boolean {
```

Comment:
The name is too generic. Looks like it should be comparing any two arrays and just so happens typescript can tell us.